zstacklib/zstacklib/utils/sanlock.py

```
@@ -219,6 +235,35 @@ def wait(_):
+
+    result = {}
+    meta_backup = linux.read_file(bk_file)
+    try:
+        for line in meta_backup:
+            line = line.strip()
+            if not line or line.startswith('#'):
+                continue
+            if line.startswith('host_id '):
+                result['host_id'] = int(line.split()[1])
+            elif line.startswith('generation '):
+                result['generation'] = long(line.split()[1])
```

Ruff (0.11.9)

Undefined name `long`

(F821)